.tests/units/fix_af_test.py

```diff
@@ -10,9 +10,15 @@
 import io
 import os
 import pysam
+import sys
 import unittest
 from dataclasses import dataclass
-from fix_af import getCaller, modifyHeader, fixFreebayes, writeNewVcf
+
+TEST_DIR = os.path.dirname(os.path.abspath(__file__))
+SCRIPT_DIR = os.path.abspath(os.path.join(TEST_DIR, "../../workflow/scripts"))
+sys.path.insert(0, SCRIPT_DIR)
+
+from fix_af import getCaller, modifyHeader, fixFreebayes, writeNewVcf  # noqa
```

Comment:
noqa?

Reply:
skip quality check for that line! Quality check would like to have it together with the other imports, but it needs to be after sys.path modification.